cross/unzip/Makefile

```diff
@@ -11,15 +11,18 @@ HOMEPAGE = http://infozip.org/UnZip.html
 COMMENT  = UnZip is an extraction utility for archives compressed in .zip format
 LICENSE  = http://infozip.org/license.html
 
-CONFIGURE_TARGET = myConfigure
+GNU_CONFIGURE = 1
+CONFIGURE_TARGET = none
 COMPILE_TARGET = myCompile
+INSTALL_TARGET = myInstall
```

Comment:
Please take opportunity to rename targets as "unzip_configure" and "unzip_install"